src/data/services/LmsApiService.js

```diff
@@ -235,12 +237,30 @@ class LmsApiService {
     return LmsApiService.apiClient().get(`${LmsApiService.lmsIntegrationUrl}/configs/?${queryParams.toString()}`);
   }
 
-  static fetchContentMetadataItemTransmission(uuid, channelCode, configId) {
-    return LmsApiService.apiClient().get(`${LmsApiService.lmsContentSyncStatusUrl}/${uuid}/${channelCode}/${configId}`);
+  static generateFormattedStatusUrl(url, currentPage, options) {
```

Comment:
Don't know if this method should be in this file, maybe we should put it in utils just so keep this to only communications with the API service?

Reply:
a fine suggestion!